.github/workflows/build.yml

```diff
@@ -48,6 +48,8 @@ jobs:
       - name: All features
         if: matrix.toolchain != '1.41.1'
         uses: actions-rs/cargo@v1
+        env:
+          RUSTFLAGS: "--cfg bench"
```

Comment:
Sorry for not being explicit; what I meant was to have if-test in github action checking for the compiler version and than skipping/running bench tests depending on the result

Reply:
I am sorry that I don't see well what you mean. Is `if: matrix.toolchain != '1.41.1'` not sufficient for `checking for the compiler version`?

Reply:
cargo downloads and compiles `criterion` if and only if `RUSTFLAGS="--cfg bench"`. So we have to specify `cfg` to use `--all-targets`

Reply:
Yes, the matrix if check is sufficient. But I do not get why we need to explicitly specify `cfg` if it should be automatically assigned by `cargo bench`

Reply:
> if it should be automatically assigned by cargo bench

It is not assigned by cargo. `test` is the only cfg they assign